package/contents/ui/config/ConfigCalendar.qml

```diff
@@ -40,6 +44,38 @@ ConfigPage {
 		text: i18n("Style")
 	}
 	ConfigSection {
+		RowLayout {
+			Layout.fillWidth: true
+			ConfigCheckBox {
+				configKey: 'monthCurrentCustomTitle'
+				text: i18n("Current Month Title:")
+			}
+			ConfigString {
+				id: monthCurrentCustomTitleFormat
+				configKey: 'monthCurrentCustomTitleFormat'
+				placeholderText: i18nc("calendar title format for current month", "MMMM d, yyyy")
+			}
+			Label {
+				text: Qt.formatDateTime(new Date(), calendarCurrentTitleDisplay)
```

Comment:
Just use `monthCurrentCustomTitleFormat.value` here instead of creating a new property.